Tests/Cosmos.Compiler.Tests.Bcl/System/Int64Test.cs

```diff
@@ -147,6 +148,32 @@ public static void Execute()
 
             ByRefTestMethod(ref value);
             Assert.IsTrue(value == 61, "Passing an Int64 by ref to a method doesn't work");
+
+            //Test StackOverflow Exceptions
+            long val3o;
+            efuse = false;
+            val3o = 1000000;
+            try
+            {
+                val3o += long.MaxValue;
+            }
+            catch (OverflowException e)
+            {
+                efuse = true;
+            }
+            Assert.IsTrue(efuse == false, "Add_Ovf for Int16 doesn't work");
+
+            efuse = false;
+            val3o = -10000;
+            try
+            {
+                val3o -= long.MaxValue;
+            }
+            catch (OverflowException e)
+            {
+                efuse = true;
+            }
+            Assert.IsTrue(efuse == false, "Sub_Ovf for Int16 doesn't work");
```

Comment:
Should be `Assert.IsTrue(efuse, ...)`.

Reply:
Thanks. Now the test fails because it doesn't throw an exception correct?

Reply:
It should be false, because if it doesn't overflow like I want it to then it's not working, i'm a little confused why you think i should do efuse not not == false.